### ISSUE TITLE
input type=button does not work with aria-pressed (IE 11)

### ISSUE THREAD

**Issue by JAWS-test** [FreedomScientific/VFO-standards-support#302]
### Summary

input type=button does not work with aria-pressed (IE 11)

1. save as HTML file:
```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="utf-8">
		<title>button</title>
	</head>
	<body>
		<span tabindex=0 role=button aria-pressed=true>1</span>
		<button aria-pressed=true>2</button>
		<input type=button aria-pressed=true value=3>
		<input type=image src=https://upload.wikimedia.org/wikipedia/commons/thumb/d/dc/Marque_number_4_black.svg/1024px-Marque_number_4_black.svg.png width=20px aria-pressed=true alt=4>
	</body>
</html>
```
2. Read with arrow keys or navigate with B or Tab key or press INS+F5

### Expected result
all buttons are output as toggle buttons

### Actual result 
* IE 11: <code>input type=button</code> with <code>aria-pressed</code> is not output at all 
* During tab navigation, the previously focused element is output.
* Problem does not occur in Chrome and Firefox
* Problem does not occur in IE 11 with NVDA
* with Inspect.exe no errors can be detected at the button

#### JAWS version and build number

JAWS 2019.1907.42
 
#### Operating System and version

Windows 8

#### Browser and version:

Chrome 76.0.3809.100
Firefox 68.0.2
Internet Explorer 11.0.9600.19431

**Comment by stevefaulkner** [comment]
Closing this issue as wontfix as its IE only
